### ISSUE TITLE
18.03.0-ce-mac59  install error

### ISSUE THREAD

**Issue by gxgeek** [docker/for-mac#2739]
```
Docker for Mac: version: 18.03.0-ce-mac59 (dd2831d4b7421cf559a0881cc7a5fdebeb8c2b98)
macOS: version 10.12.4 (build: 16E195)
logs: /tmp/3AFCD9BF-082E-41DB-8494-BE3BE8905893/20180328-103705.tar.gz
failure: com.docker.vpnkit is not running
[ERROR]  vpnkit
         Unexpected error ((Failure
  "Error connecting socket to 9p endpoint unix:/Users/gaoxing/Library/Containers/com.docker.docker/Data/s51: Unix.Unix_error(Unix.ECONNREFUSED, \"connect\", \"\")")) connecting to /Users/gaoxing/Library/Containers/com.docker.docker/Data/s51
         com.docker.vpnkit is not running
         vpnkit check failed with: Unix.Unix_error(Unix.ECONNREFUSED, "connect", "")
[OK]     vmnetd
[OK]     dns
[ERROR]  driver.amd64-linux
         com.docker.driver.amd64-linux is not running
[OK]     virtualization VT-X
[OK]     app
[OK]     moby
[OK]     system
[OK]     moby-syslog
[OK]     kubernetes
[OK]     files
[OK]     env
[OK]     virtualization kern.hv_support
[ERROR]  osxfs
         com.docker.osxfs is not running
[OK]     moby-console
[OK]     logs
[ERROR]  docker-cli
         Connection refused (ECONNREFUSED) connecting to /var/run/docker.sock: check if service is running
         Connection refused (ECONNREFUSED) connecting to /Users/gaoxing/Library/Containers/com.docker.docker/Data/s60: check if service is running
         docker ps failed
[OK]     disk
```

**Comment by JuliaRechkunova** [comment]
@gxgeek Reset to factory defaults helped me to fix it

**Comment by gtardif** [comment]
This was due to a problem when migrating straight from 17.09 to 18.03. 
It should be fixed with the latest Docker for Mac 18.03.0-mac60 Stable release (you can download at https://download.docker.com/mac/stable/Docker.dmg)
